include/networkmanager.h

```diff
@@ -6,7 +6,7 @@
 
 #include <QDnsLookup>
 #include <QNetworkAccessManager>
-#include <QTcpSocket>
+#include <QtWebSockets/QWebSocket>
```

Comment:
where did TCP go

Reply:
bye

Reply:
it's probably being included transitively by another include, I wouldn't worry about it too much.